src/pipeline_migration/actions/migrate/cli.py

```diff
@@ -17,9 +72,30 @@ def arg_type_upgrades_file(value: str) -> Path:
     return p
 
 
+def arg_type_pipeline_file(value: str) -> str:
+    p = arg_type_upgrades_file(value)
```

Comment:
`arg_type_upgrades_file` will throw a confusing error if the path doesn't exist:

```
$ pmt migrate --pipeline-file foo
pmt migrate: error: argument -p/--pipeline-file: Upgrades file foo does not exist.
```

Should we rename it to a generic `arg_type_file` and make the error message generic too?